.github/workflows/ci.yml

```
@@ -7,6 +7,7 @@ jobs:
       fail-fast: false
       max-parallel: 20
       matrix:
+        branch: [v1.4.8, v1.6.2]
```

```suggestion
        branch: [v1.4.8, v1.6.4]
```
Latest in the 1.6 releases is 1.6.4; not sure it matters that much, but similarly 1.4.8 is the latest in the 1.4 releases.

---

Actually, I don't think the `branch` entry in the matrix is in any way used since nim-dagger was switched to a nimbus-build-system setup.

There is only one hit in `ci.yml` for `branch` and it's that L10.